boards/arm/mimxrt685_evk/mimxrt685_evk_cm33_defconfig

```diff
@@ -14,8 +14,7 @@ CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_CORTEX_M_SYSTICK=y
 CONFIG_GPIO=y
 CONFIG_PINMUX=y
-CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=48000000
-
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=250105263
```

Comment:
This is a strange number, is it correct?

Reply:
checked the sdk code, the value is correct

Reply:
The value is derived from the main PLL PFD.
24 (SYSOSC) * 22 (SYSPLL) * 18 / 19 (PFD0) / 2 (CORE_DIV)